ocdskingfisherviews/refresh_views.py

```diff
@@ -45,6 +46,11 @@ def refresh_views(engine, viewname, remove=False):
             with engine.begin() as connection:
                 schema_name = engine.dialect.identifier_preparer.quote('view_data_' + viewname)
                 connection.execute('SET search_path = {}, public;'.format(schema_name))
+
+                if tables_only:
+                    statement_part = re.sub('^create view', 'create table', statement_part, flags=re.M | re.I)
+                    statement_part = re.sub('^drop view', 'drop table', statement_part, flags=re.M | re.I)
```

Comment:
Why is multiline `re.M` needed?

Reply:
In mulitline mode `^` means either the start of the string or after a newline [see docs.](https://docs.python.org/3/library/re.html#regular-expression-syntax)
I wanted to match in those circumstances.  Without multiline mode I think `^` only matches the start of the whole string.

Reply:
Aha, right!